### ISSUE TITLE
Further enhancements to menu tooltips/keybinds

### ISSUE THREAD

**Issue by aubergine10** [CitiesSkylinesMods/TMPE#720]
While testing #656 I noticed some issues with tooltips and keybinds, I think maybe a separate PR could imrpove things a little if time permits...

I just added a keybind to Parking Restrictions for first time and got this:

![image](https://user-images.githubusercontent.com/1386719/74893646-ca1e6700-5384-11ea-8574-8eaa02e3d4fa.png)

Also, I think we should change where tool-select keybind is shown, if that's possible? For example, this is what Priority Signs tooltip currently looks like:

![image](https://user-images.githubusercontent.com/1386719/74893759-04880400-5385-11ea-919c-cc200015fcb7.png)

It's difficult to see clearly what is going on there.

I would suggest cleaning it to something like:

>Priority Signs (Shift + P)
>  
> When tool is active, you can:
> Shift+Click: ...
> Ctrl+Click: ...
> Ctrl+Shift+Click: ...

Also, is it possible to have keybind (ie, the shortcut key strings, _not_ their effect) in different colour such as yellow?

Following tools are still missing keybinds:

* Timed Traffic Lights and Manual Traffic Lights
    * I hope we can one day merge all traffic lights in to one completely refactored tool
    * _So ignore these for now_
* Disable Despawning
   * Keybind would toggle state
* Vehicle Restrictions
* Clear Traffic
    * _Doesn't need a keybind IMO_

**Comment by krzychu124** [comment]
> Also, is it possible to have keybind (ie, the shortcut key strings, not their effect) in different colour such as yellow?

Only when we separate keybind from actual translation. I mentioned about that somewhere (github/discord), don't remember.

With current implementation we can't really change keybind without additional help of translators (you have to alter all languages), not to mention that translation string change is pretty hard to spot. 
I couldn't find a good way to notify other translators about change, because notifications works only for new strings/languages.

_I would love to see OSD in action instead of boring tooltips_ 😄

**Comment by aubergine10** [comment]
Another feedback, the left/right padding on tooltips seems overly huge:

![image](https://user-images.githubusercontent.com/1386719/74897047-c216f500-538d-11ea-93ad-2766441d33e0.png)

**Comment by kvakvs** [comment]
@kianzarrin there was this https://github.com/CitiesSkylinesMods/TMPE/pull/500
and this https://github.com/CitiesSkylinesMods/TMPE/pull/421
I will need to take pieces of that work, plus create compact buttons icons, and we'll be good on keybinds.

**Comment by aubergine10** [comment]
Excluding additional keybinds, #868 fixes this issue #720. Additional keybinds stuff can get added when individual tools are updated in future PRs.

**Comment by aubergine10** [comment]
doh
